src/main/java/cz/cvut/kbss/study/rest/ActionHistoryController.java

```diff
@@ -54,7 +54,7 @@ public void create(@RequestBody ActionHistory actionHistory) {
         }
     }
 
-    @PreAuthorize("hasRole('" + SecurityConstants.ROLE_ADMIN + "')")
+    @PreAuthorize("hasAuthority('" + SecurityConstants.administrator + "')")
```

Comment:
Why do you like to call it hasAuthority, instead of hasRole?

Reply:
hasRole() and hasAuthority() do the same thing, but the key difference is that hasRole() automatically adds the "ROLE_" prefix when checking the user's role. For example, hasRole("ADMIN") will check if the user has the role ROLE_ADMIN. Since we don't follow this convention, I replaced it with hasAuthority(), which does the same thing without adding the prefix.